crates/python/src/qpu/client.rs

```diff
@@ -127,6 +128,25 @@ impl PyQcsClient {
                 .map_err(LoadError::to_py_err)?,
         })
     }
+
+    async fn load(profile_name: Option<String>, use_gateway: Option<bool>) -> PyResult<Self> {
+        let config = match profile_name {
+            Some(profile_name) => ClientConfiguration::load_profile(profile_name).await,
+            None => ClientConfiguration::load_default().await,
+        };
+
+        let client = config
+            .map(Qcs::with_config)
+            .map_err(LoadError)
+            .map_err(ToPythonError::to_py_err)?;
+
+        let client = match use_gateway {
+            None => client,
+            Some(use_gateway) => client.with_use_gateway(use_gateway),
+        };
+
+        Ok(Self(client))
+    }
```

Comment:
Could you use `py_function_dual` for this as well?

Reply:
With some modification, yes - need to add a branch for handling between `#[staticmethod]` and `#[pyfunction]`

Reply:
I tried this, but ran into an issue: since the outer `impl` block has a proc macro on it, things got complicated:

```rs
#[pymethods]
impl PyClient {
    #[staticmethod] // <- this is fine
    #[args("/", profile_name = "None", use_gateway = "None")]
    /* async fn load ... */

    py_function_sync_async! {
        #[staticmethod] // <- error, `staticmethod` not found in scope
        #[args("/", profile_name = "None", use_gateway = "None")]
        /* async fn load ... */
    }
}
```